app/models/comment.php

```diff
@@ -20,17 +20,116 @@ public static function getComments($offset, $limit, $id)
         $db = DB::conn();
         $rows = $db->rows("SELECT * FROM comment 
                         WHERE thread_id = ? ORDER BY created ASC LIMIT {$offset}, {$limit}", array($id));
```

Comment:
I think it's better to put placeholders for the values of $offset and $limit  :bow: